mesa/examples/advanced/epstein_civil_violence/app.py

```diff
@@ -8,10 +12,9 @@
 )
 
 COP_COLOR = "#000000"
-AGENT_QUIET_COLOR = "#648FFF"
-AGENT_REBEL_COLOR = "#FE6100"
-JAIL_COLOR = "#808080"
-JAIL_SHAPE = "rect"
+QUIET_COLOR = "#648FFF"
+ACTIVE_COLOR = "#FE6100"
+ARRESTED_COLOR = "#808080"
```

Comment:
I would make this a dict
```Python
agent_colors = {
    CitizenState.ACTIVE: "#FE6100",
    CitizenState.QUIET: "#648FFF",
    CitizenState.ARRESTED: "#808080"
}
```
then you can do fast lookups instead of matching (which is quite slow for every agent for every step).